components/UnifiedDocFeed/FeedMenu/FeedMenuMobileScrollControls.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState, useEffect } from "react";
+import useEffectForHorizontalScroll from "../utils/useEffectForHorizontalScroll";
+import { css, StyleSheet } from "aphrodite";
+import icons from "~/config/themes/icons";
+
+const FeedMenuMobileScrollControls = ({ tabsContainerRef, viewportWidth }) => {
+  const [showMobileLeftScroll, setShowMobileLeftScroll] = useState(false);
+  const [showMobileRightScroll, setShowMobileRightScroll] = useState(false);
+  const [showMobileScrollNav, setShowMobileScrollNav] = useState(false);
+
+  useEffect(() => {
```

Comment:
same here. usage of useEffect within useEffect looks very weird.

Reply:
The implementation is solid but you questioning of it made me feel that it is not clear so I renamed things and aliased.

This is responsible for determining whether or not to display the mobile scroll (see right arrow)

![image](https://user-images.githubusercontent.com/802819/187297704-c13eae32-98a1-4d9f-b7b2-16710228ed8e.png)